lib/helpers.js

```diff
@@ -207,11 +213,11 @@ _beforeEach.givenLoggedInUserWithRole = function(credentials, role, optionalHand
 }
 
 _beforeEach.givenAnUnauthenticatedToken = function(attrs, optionalHandler) {
-  _beforeEach.givenModel('accessToken', attrs, optionalHandler);
+  _beforeEach.givenModel('AccessToken', attrs, optionalHandler);
```

Comment:
@justin-lau `this` doesn't point to the test object. `'AccessToken'` is the key of  `options`, it will be use to retrieve the configured model name in `_beforeEach#givenModel()`.